sig-testing/README.md

```diff
@@ -65,6 +57,10 @@ Kubernetes IN Docker. Run Kubernetes test clusters on your local machine using D
   - sigs.k8s.io/kind bi-weekly meeting: [Mondays at 11:00 PT (Pacific Time)](https://zoom.us/j/960461819) (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
     - [Meeting notes and Agenda](https://docs.google.com/document/d/1b9Ppm7ZT_tMWRs5Ph1zGJJKb5nF9c3ZHzMwg1olJIrc/edit).
     - [Meeting recordings](https://bit.ly/k8s-sig-testing-videos).
+### kubetest2
+Kubetest2 is intended to be the next significant iteration of kubetest
```

Comment:
we should include kubetest, but note that we're deprecating it.

Reply:
So I'm tempted to either:
- lump that in with kubetest2 and call it the kubetest subproject
- make a 'legacy-test-infra' subproject, and bin things like bootstrap and kubetest under it

Reply:
I opted for @BenTheElder's suggestion, leaving this named as kubetest2 but noting kubetest will be deprecated